ui2/src/components/app/Container.jsx

```diff
@@ -1,44 +1,36 @@
 import React, { useState } from 'react';
 import { Breadcrumb } from 'antd';
 import { Link } from 'react-router-dom';
+import { parseSearchUrl } from '../../utils/parse';
```

Comment:
use url-parse from npm

Reply:
I tried but they don't have a function that returns an ordered array of the search keys, I'll try to find another solution